cvat/settings/base.py

```diff
@@ -68,14 +67,14 @@ def add_ssh_keys():
         fcntl.flock(pid, fcntl.LOCK_EX)
         try:
             add_ssh_keys()
-            keys = subprocess.run(['ssh-add -l'], shell = True,
```

Comment:
What is a reason to remove `shell = True`?

Reply:
fixing bandit issue